integration-tests/features/User.feature

```diff
@@ -6,6 +6,12 @@ Feature: Features related to User Management
     When I make a POST to the function at '/organisations/validOrg/users'
     Then I should get a status code 201
     And the response should be a superset of all the keys and values set from 'newUserRequest'
+    
+  @getUserFromOrg @createdUserInOrg
+  Scenario: GET to /users/{userId} with valid userId returns user
+   When I make a GET to the function at '/organisations/validOrg/users/newUser'
+   Then I should get a status code 200
+   And the response should be a superset of all the keys and values set from 'newUserRequest'
```

Comment:
The problem we have here now is that we have integration tests that now depend on one another running in a certain order. As scenarios will get more complex we generally want to avoid this. One way of resolving this is to add the steps for creating the above user within the feature itself so:

```suggestion
    When I make a POST to the function at '/organisations/validOrg/users'
    Then I should get a status code 201
    And the response should be a superset of all the keys and values set from 'newUserRequest'
    
  @getUserFromOrg @createdUserInOrg
  Scenario: GET to /users/{userId} with valid userId returns user
   Given I generate a json payload called 'newUserRequestWithSpecificId'
   And I make a POST to the function at '/organisations/validOrg/users'
   And the response should be a superset of all the keys and values set from 'newUserRequestWithSpecificId'
   When I make a GET to the function at '/organisations/validOrg/users/newUser'
   Then I should get a status code 200
   And the response should be a superset of all the keys and values set from 'newUserRequestWithSpecificId'
```